plugins/scaffolder-backend/src/scaffolder/actions/types.ts

```diff
@@ -45,6 +46,20 @@ export type ActionContext<Input extends JsonObject> = {
    * This will only ever be true if the actions as marked as supporting dry-runs.
    */
   isDryRun?: boolean;
+
+  /**
+   * The user which triggered the action.
+   */
+  user?: {
```

Comment:
Why redefine this type: `TaskSpec['user']` looks the same, can/should we extract a new type from the TaskSpec or create a new type for the task spec user.

Reply:
It's fine by me, Used the same approach as it was in Backstage

![Screenshot 2022-09-16 at 11 21 27](https://user-images.githubusercontent.com/2265094/190604245-a5231281-39f7-447c-98ad-18db726924e8.png)

I'm not sure if I have to refactor the code in multiple modules in scope of this story.